java-management/workshops/tenancy/manifest.json

```diff
@@ -49,6 +49,10 @@
             "title": "Lab 9: Use Java Download",
             "filename": "../../java-download/java-download.md"
         },
+        {
+            "title": "Lab 10: JMS Kubernetes Support",
```

Comment:
The workshop title should be indicate what the user will be doing. Consider something along the lines of "Lab 10: Install Management Agent in your Kubernetes Cluster" . Rename the file name and directories accordingly.